### PR TITLE
Profile Community Update

### PR DESCRIPTION
**Pull request by katagatame**
This PR resolves the following issues:
- #62 
- #63

### CODE REVIEW

File: src/commands/assignment/CreatePost.ts [hunk 1]
```diff
@@ -21,13 +22,13 @@ export default class CreatePost extends Command {
 		const embed: RichEmbed = new RichEmbed()
 			.setColor(Constants.embedColor)
 			.setTitle(`Role Assignment`)
-			.setDescription(`Please react to this post with your main console.\n\n`);
+			.setDescription(`Please react to this post with your main platform.\n\n`);
```
Comment (D0cR3d) [inline src/commands/assignment/CreatePost.ts]: Shouldn't this just say "Please react with each of the platforms you will be playing Destiny on."

File: src/commands/profile/Me.ts [hunk 2]
```diff
@@ -71,10 +64,9 @@ export default class Me extends Command {
...
 				}
 
-				if (error) {
-					message.channel.send(errorMessage);
-					return message.channel.stopTyping();
-				}
+				if (error)
+					return message.channel.send(errorMessage);
+
 				if (/pc|ps|psn/i.test(args[1])) {
```
Comment (D0cR3d) [inline src/commands/profile/Me.ts]: PC needs to be separated out from here, right?

Comment (katagatame) [inline src/commands/profile/Me.ts]: no, currently they pass through the same regex.

File: src/commands/assignment/CreatePost.ts (hunk 1 above)
Comment (katagatame) [inline src/commands/assignment/CreatePost.ts]: will update.

**Pull request merged**